cookbooks/bcpc-hadoop/recipes/zookeeper_impl.rb

```diff
@@ -4,7 +4,6 @@
 ::Chef::Resource::File.send(:include, BCPC::Utils)
 
 include_recipe 'bcpc-hadoop::zookeeper_packages'
-include_recipe 'bach_krb5::keytab_directory'
```

Comment:
Isn't this required?

Reply:
See circular dependency issue elsewhere. I will leave this recipe in `barch_krb5` for now but we can open a future PR if we want to refactor the bach_krb5 cookbook.